mandos-go/json/write/writeAccount.go

```diff
@@ -5,7 +5,8 @@ import (
 	oj "github.com/ElrondNetwork/arwen-wasm-vm/mandos-go/orderedjson"
 )
 
-func accountsToOJ(accounts []*mj.Account) oj.OJsonObject {
+// AccountsToOJ converts a mandos-format account to an ordered JSON representation.
+func AccountsToOJ(accounts []*mj.Account) oj.OJsonObject {
```

Comment:
Why uppercase "A"? Isn't the standard to have all functions starting with lowercase letter in golang?

Reply:
`UpperCamelCase` = public
`lowerCamelCase` = private in Go